### DISCUSSION TITLE
ZSA Voyager custom mapping, achordion and gaming layers custom_config.h

### DISCUSSION THREAD

**Discussion by UnitRoot22** [manna-harbour/miryoku#362] [Show and tell]
[Link](https://github.com/UnitRoot22/qmk_userspace/blob/main/users/miryoku/custom_config.h).

Still new to QMK and split keyboards, probably have done some things incorrectly, but: 

- QMK
- ZSA Voyager
-  Achordion for better feeling home row mods.
- Changed mapping to use the 2 thumb keys with shifted layout.
- Backlight colors for each layer
- Gaming layer that can be accessed from the base layer with the extra unused voyager keys
- Gaming numbers layer on top of gaming layer for 0-9 and some extra keys.

**Comment by manna-harbour** [comment]
Thanks for sharing your custom config!

I'd generally suggest putting the extra keys into a pre-layer per-keyboard mapping, so the layers can be shared between keyboards, but if you only use one keyboard this works too!
